src/settingsui.ts

```diff
@@ -270,6 +320,14 @@ class SettingElem {
 				this.updateMethod = 'onchange';
 				break;
 			}
+			case 'color':
```

Comment:
nice. you can also add a keybind type setting, if you want to.
it would store a string/object of a key, on click captures a new key and escape cancels the setting of a new keybind. would be useful for example for the keystrokes userscript.